### PR TITLE
Add op (std, std.dim, std.correction) | feat(torchlib)

### PR DESCRIPTION
**Pull request by titaiwangms**
Add std, std.dim, and std.correction

### CODE REVIEW

File: tests/function_libs/torch_lib/ops_test_data.py [hunk 1]
```diff
@@ -2300,6 +2327,8 @@ def _where_input_wrangler(
 ops_test_common.duplicate_opinfo(OPS_DB, "ops.aten._softmax", ("ops.aten._softmax_half",))
 ops_test_common.duplicate_opinfo(OPS_DB, "round", ("round_decimals",))
 ops_test_common.duplicate_opinfo(OPS_DB, "squeeze", ("squeeze_dim",))
+ops_test_common.duplicate_opinfo(OPS_DB, "std", ("std_dim", "std_correction"))
+ops_test_common.duplicate_opinfo(OPS_DB, "var_mean", ("var_mean_dim", "var_mean_correction"))
```
Comment (justinchuby) [inline tests/function_libs/torch_lib/ops_test_data.py:2331]: Curious why duplicating var_mean was needed?

Comment (justinchuby) [inline tests/function_libs/torch_lib/ops_test_data.py:2331]:
```suggestion
```

looks duplicated

**Review by justinchuby — APPROVED** [review]
Thank you!

File: tests/function_libs/torch_lib/ops_test_data.py (hunk 1 above)
Comment (titaiwangms) [reply tests/function_libs/torch_lib/ops_test_data.py:2326]: Fix

**Pull request merged**